FileShareService.DesktopClient.Core/Jobs/ErrorDeserializingJobsJob.cs

```diff
@@ -7,6 +9,13 @@ public class ErrorDeserializingJobsJob : IJob
     {
         public Exception Exception { get; }
 
+        public List<string> ErrorMessages { get; set; } = new List<string>();
+
+        public ErrorDeserializingJobsJob()
```

Comment:
I think we have the potential for a null ref error when `ErrorDeserializingJobsJob` is constructed using the default constructor. In this scenario `Exception` will be null and `DisplayName` will throw a null ref exception. Could you please address this.

Reply:
Fixed. Now display name is a text, which will be used as header content of error summary panel. And, the exception message is added in ErrorMessages property.